userspace/su/jni/su.c

```diff
@@ -1,10 +1,118 @@
+#include <stdio.h>
+#include <string.h>
 #include <unistd.h>
 #include <stdlib.h>
+#include <stdbool.h>
 #include <sys/prctl.h>
 
-int main(){
+#define CMD_GRANT_ROOT 0
+
+#define CMD_BECOME_MANAGER 1
+#define CMD_GET_VERSION 2
+#define CMD_ALLOW_SU 3
+#define CMD_DENY_SU 4
+#define CMD_GET_ALLOW_LIST 5
+#define CMD_GET_DENY_LIST 6
+
+#define VERSION "8"
```

Comment:
Get version from prctl

Reply:
Sorry, I just forgot to remove this macro. It already use prctl to get version.